src/Equinox.MessageDb/MessageDbClient.fs

```diff
@@ -98,23 +115,17 @@ type MessageDbClient(source: CancellationToken -> Task<NpgsqlConnection>) =
 
         cmd
 
-    member __.WriteMessages(streamName, events : _ array, version : int64, ct) =
-        task {
-            try
-                use! conn = source ct
-                use transaction = conn.BeginTransaction()
-                use batch = new NpgsqlBatch(conn, transaction)
-
-                let mutable expectedVersion = version
-
-                for event in events do
-                    batch.BatchCommands.Add(__.PrepareWriteCommand(streamName, expectedVersion, event))
+    member __.WriteMessages(streamName, events : _ array, version : int64, ct) = task {
```

Comment:
__

Reply:
What would you like it to be?

Reply:
`member x.WriteMessages(streamName, events : _ array, version : int64, ct) = task {`

(or maybe `client.` ?)

Reply:
client sounds sensible.